### DISCUSSION TITLE
Proxy configuration for Webhooks

### DISCUSSION THREAD

**Discussion by Nankh** [jreleaser/jreleaser#1453] [Q&A] [answered]
Hey guys !

thanks for the awesome work !

i'm just having a pb when trying to configure a webhook through jreleaser. 
I have proxy issue when trying to reach teams notification webhook.

We gave a try with the -Dhttp.proxy options, but no success so far.
Do you have an idea how we can handle this ?

Many thanks !

**Comment by aalmiray** [comment, marked as answer]
AFAICT standard System properties for defining proxy settings should be honored. If they are no then it’s likely we need to fix something. 

Which mode (cli, maven, gradle, ant) are you using to invoke JReleaser?

**Reply by Nankh** [reply]
you're right ! thks 👍 

i've overrided the Maven_OPTS and it's now working fine (i was using a different configuration before)

**Reply by aalmiray** [reply]
Great!
